openedx/adg/admin_dashboard/performance/constants.py

```diff
@@ -14,3 +14,5 @@
     _('Completion Date'),
 ]
 ALL_BUSINESS_UNITS = _('All Business Units')
+
+PAGINATE_COURSE_LIST_BY = 10
```

Comment:
@annas-usman - Just a suggestion, can we update this variable name to something like 
`COURSE_LIST_PAGE_SIZE`
It seems more meaningful rather than use paginate word in variable name